<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyPostsDialog</class>
 <widget class="QDialog" name="MyPostsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Posts</string>
  </property>

  <!-- Main Frame with gradient background -->
  <widget class="QWidget" name="mainFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QWidget#mainFrame {
    background-color: qlineargradient(
        spread:reflect,
        x1:0, y1:0,
        x2:1, y2:1,
        stop:0 rgba(148, 4, 3, 255),
        stop:1 rgba(239, 174, 20, 255)
    );
}
    </string>
   </property>

   <!-- Posts List -->
   <widget class="QListWidget" name="myPostListWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>560</width>
      <height>250</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; border-radius: 8px; font: 11pt "Arial";</string>
    </property>
   </widget>

   <!-- Edit Button -->
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green; color: white; border-radius: 10px; font: 11pt "Arial";</string>
    </property>
   </widget>

   <!-- Delete Button -->
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red; color: white; border-radius: 10px; font: 11pt "Arial";</string>
    </property>
   </widget>

   <!-- Close Button -->
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>290</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray; color: white; border-radius: 10px; font: 11pt "Arial";</string>
    </property>
   </widget>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>